### PR TITLE
updated CPU and Registers,cache,and ram

### PR DESCRIPTION
**Pull request by glenntu15**
Updated Readme.md. I changed Registers and ram to registers cache and ram.  I moved this section after the CPU section. I added descriptions of cache and a word or two about cores.

### CODE REVIEW

**Comment by shhossain** [comment]
Will be merged after few hours.

File: readme.md [hunk 1]
```diff
@@ -95,11 +88,7 @@ Non-primitive data types are also known as reference data types. They are create
 - [Class](Data%20Types/readme.md#class)
 
 ## [Statements and Functions](#statement-and-function)
-In computer programming, a statement is a syntactic unit of an imperative programming language that expresses some action to be carried out. A program written in such a language is formed by a sequence of one or more statements. A statement may have internal components (e.g., expressions).
```
Comment (shhossain) [inline readme.md:98]: Why are you removing this?

File: readme.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
 - [Electronic Computing](#electronic-computing)
 - [Boolean Logic](#boolean-logic)
 - [Number Systems](#number-systems)
-- [Registers and RAM](#registers-and-ram)
```
Comment (shhossain) [inline readme.md:9]: TOC should have a section on "Registers cache and ram". Add "[Registers, Cache and RAM](#registers-CACHE-and-ram)" in it.

File: readme.md [hunk 3]
```diff
@@ -50,26 +50,19 @@ Number systems are a mathematical system for expressing numbers. A number system
...
 
 ## [Instruction and Program](#instruction-and-program)
 In computer science, an instruction is a single operation of a processor defined by the processor instruction set. And a computer program is a list of instructions that tell a computer what to do. Everything a computer does is done by using a computer program. Programs stored in the memory of a computer ("internal programming") let the computer do one thing after another, even with breaks in between.
 
 ## [Programming Language](#programming-language)
 A programming language is any set of rules that converts strings, or graphical program elements in the case of visual programming languages, to various kinds of machine code output.[citation needed] Programming languages are one kind of computer language, and are used in computer programming to implement algorithms.
 
-Some Languages Include:
```
Comment (shhossain) [inline readme.md:66]: Why are you removing this?

**Review by shhossain — CHANGES_REQUESTED** [review]
Resolve three problems.

File: readme.md (hunk 3 above)
Comment (glenntu15) [reply readme.md:66]: I apparently did not have a current version of the readme

Comment (shhossain) [reply readme.md:66]: Fork the latest repo and resubmit the pull request. Or you can fix it any other way.

File: readme.md (hunk 1 above)
Comment (glenntu15) [reply readme.md:98]: How do I withdraw the pull request.  I don't have time today.  I'll need to start over.

Comment (shhossain) [reply readme.md:98]: You can close the pull request.

**Comment by glenntu15** [comment]
closed I saved my text and will try again another time

**Comment by shhossain** [comment]
🆗

**Pull request closed without merging**